resources/skins.citizen.styles/components/Footer.less

```diff
@@ -22,7 +22,6 @@
 		display: flex;
 		gap: var( --space-md ) var( --space-xxl );
 		justify-content: space-between;
-		padding: var( --space-md ) 0;
```

Comment:
This is intended to keep a consistent and symmetrical spacing for content and bottom